client/packages/coldchain/src/Equipment/api/hooks/document/useAssets.ts

```diff
@@ -14,6 +14,7 @@ export const useAssets = () => {
       { key: 'typeId', condition: 'equalTo' },
       { key: 'isNonCatalogue', condition: '=' },
       { key: 'store' },
+      { key: 'functionalStatus', condition: 'equalTo' },
```

Comment:
Not related to this change, but IMHO we shouldn't fetch query params in the api hooks. This should be done closer to the page where it is used.

Reply:
I agree 100%